dist/view.js

```diff
@@ -298,4 +299,43 @@ function reactVirtualizedToReactVirtualized(view) {
     rowCount: rowCount,
     columnWidth: columnWidth
   };
+}
+
+/**
+  * Ensure dates and times in resources are in ISO format.
+  * @param {resource}
+  * @return {resource} with formatted dates and times
+  */
+function normalizeDateAndTime(resource) {
+  if (resource._values) {
+    var dateFields = [],
+        timeFields = [],
+        dateTimeFields = [];
+
+    resource.schema.fields.forEach(function (field, idx) {
+      if (field.type === 'date') {
+        dateFields.push(idx);
+      } else if (field.type === 'time') {
+        timeFields.push(idx);
+      } else if (field.type === 'datetime') {
+        dateTimeFields.push(idx);
+      }
+    });
+
+    resource._values.forEach(function (entry) {
+      dateFields.forEach(function (fieldIdx) {
+        var date = entry[fieldIdx].toString();
+        date = new Date(date.substring(0, 28));
+        entry[fieldIdx] = date.toISOString().substring(0, 10);
```

Comment:
@zelima the reason behind this is we are getting dates with times as: `Mon Feb 22 2016 00:00:00 GMT+0600 (+06)` which includes timezone of the machine. If we do `.toISOString()` it will convert it to UTC and date will change to the previous day. So we are first converting it to string and cutting substring without timezone (having just GMT).